src/Hazelcast.Net/Serialization/InvalidPortableFieldException.cs

```diff
@@ -60,15 +62,18 @@ public InvalidPortableFieldException(string message, Exception innerException)
             : base(message, innerException)
         { }
 
+#if !NET8_0_OR_GREATER
         /// <summary>
         /// Initializes a new instance of the <see cref="InvalidPortableFieldException"/> class with serialized data.
         /// </summary>
         /// <param name="info">The <see cref="SerializationInfo"/> that holds the serialized object data
         /// about the exception being thrown.</param>
         /// <param name="context">The <see cref="StreamingContext"/> that contains contextual information
         /// about the source or destination.</param>
+        [Obsolete("This constructor is obsolete due to BinaryFormatter being obsolete. Use the constructor without this parameter.")]
```

Comment:
why don't you totally remove this constructor also for .net 7 and lower?

Reply:
It works on lower targets. I didn't want to break it suddenly.

Reply:
Is this public API?

Reply:
Some of the exceptions are public. While refactoring, I didn't separate them.